assets/model_monitoring/components/data_drift/data_drift_signal_monitor/spec.yaml

```diff
@@ -4,7 +4,7 @@ type: pipeline
 name: data_drift_signal_monitor
 display_name: Data Drift - Signal Monitor
 description: Computes the data drift between a baseline and production data assets.
-version: 0.3.46
+version: 0.3.47
```

Comment:
shouldn't weekly DRI to upgrade the versions?

Reply:
We want to do a hotfix because there is a Css ticket. I do not think there will be more PRs this week so we can skip weekly release